docs/notebooks/WMD_tutorial.py

```diff
@@ -0,0 +1,329 @@
+
+# coding: utf-8
+"""
+WMD_tutorial.py
+===================
+
+
+.. _general_examples:
+
+General examples
+----------------
+
+General-purpose and introductory examples from the sphinx-gallery
+"""
```

Comment:
If you want I can generate the py files afresh from the notebooks and edit them again. So that in each commit you can clearly see all the changes I have made to each of the py files.